bots/wangjia/wangjia/spiders/tedian.py

```diff
@@ -63,6 +63,6 @@ def parse(self, response):
             item['experience_status'] = get_content(scoreInfo[3].xpath('text()').extract())
 
             impress_info = comment_info.xpath('./dl[@class="impression"]/dd//span')
-            item['impression'] = '\001'.join([get_trunk(impress) for impress in impress_info.xpath('text()').extract()])
+            item['impression'] = '#'.join([get_trunk(impress) for impress in impress_info.xpath('text()').extract()])
```

Comment:
我一开始也这么想的，但是在数据库中的形式是一个 “口” 字符。
因此，为了方便老王做数据处理，就把两个连接符都统一换成了“#”。

2016-03-21 15:09 GMT+08:00 Zacky <notifications@github.com>:

> In bots/wangjia/wangjia/spiders/tedian.py
> <https://github.com/merryChris/blotus/pull/19#discussion_r56787115>:
>
> > @@ -63,6 +63,6 @@ def parse(self, response):
> >              item['experience_status'] = get_content(scoreInfo[3].xpath('text()').extract())
> >
> >              impress_info = comment_info.xpath('./dl[@class="impression"]/dd//span')
> > -            item['impression'] = '\001'.join([get_trunk(impress) for impress in impress_info.xpath('text()').extract()])
> > +            item['impression'] = '#'.join([get_trunk(impress) for impress in impress_info.xpath('text()').extract()])
>
> 看起来还是'\001'会好一点吧？
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/merryChris/blotus/pull/19/files/b03f60ee7e929d7c60558a481469f962760d459b#r56787115>
>